.circleci/config.yml

```diff
@@ -39,6 +42,7 @@ jobs:
       - run: yarn tsc
       - run: yarn test:all --coverage --runInBand
       - run: '[ $COVERALLS_REPO_TOKEN ] && yarn coveralls < demo/coverage/lcov.info || true'
+      - *persist_to_workspace
```

Comment:
What are we persisting from the `test` job?

Reply:
We were missing the `demo/coverage` folder on deploy. See https://github.com/zendeskgarden/react-components/tree/gh-pages

Reply:
Aw, forgot we were publishing those. 👍